src/base.js

```diff
@@ -94,6 +95,8 @@ export const concat = function(...args) {
   return new AnimatedConcat(args.map(adapt));
 };
 
+export const arrayFrom = animatedArrayFrom;
```

Comment:
I'm rather into following convention like below: 
```
export const arrayFrom = function(arrayFrom) {
	return new AnimatedArrayFrom(adapt(arrayFrom))
}
```

Reply:
I'll not be able to reuse this node if I use adapt here. Adapt will always create new node for regular values (with different id on each re-render).

Reply:
Ok, right. But maybe concern using `function` instead of exporting class

Reply:
Or not export it in `base.js` at all.

Reply:
good point, moved: https://github.com/kmagiera/react-native-reanimated/pull/110/commits/544519134b4ae0e220a754ba00be025fee1cf66c#diff-90e18a8032bc64f5e4865642e05ef908R51